### ISSUE TITLE
Ability to remove "clauses" from SQL builder

### ISSUE THREAD

**Issue by bluesnowman** [spadefoot/kohana-orm-leap#36]
See, http://dev.kohanaframework.org/issues/3495

**Comment by taai** [comment]
Yeah, I have also needed this. But only for the `limit()`. What I did - I created a `nolimit()` method. I think that it will not be too hard to remove the LIMIT and ORDER BY, but WHERE get's more complex.
Maybe some day...

**Comment by bluesnowman** [comment]
Rather than have a 'nolimit()' method, we could just accept a NULL value which removes the limit.  The same could be done with the offset method.

**Comment by bluesnowman** [comment]
Just added a new method to builder classes to reset them.
